cliqz.cfg

```diff
@@ -18,6 +21,25 @@ let extensionUrls = {
 
 let prefs = Cc["@mozilla.org/preferences-service;1"].getService(Ci.nsIPrefBranch);
 
+function verifyAddons() {
+  Cu.import("resource://gre/modules/addons/XPIProvider.jsm");
+  let versionChecker = Cc["@mozilla.org/xpcom/version-comparator;1"]
+                         .getService(Components.interfaces.nsIVersionComparator);
+  let distributionVersion = prefs.getCharPref("distribution.version");
+
+  let lastDistributionVersion;
+  try {
```

Comment:
the pref API is quite strange - if the pref is not there or not ready it simply throws an exception :/